vim/syntax/replesent.vim

```diff
@@ -0,0 +1,7 @@
+" Vim syntax file
+" Language: REPLesent
+" Latest Revision: 2016-05-17
+" Depends on: https://github.com/derekwyatt/vim-scala
+
+syntax include @Scala syntax/scala.vim
+syntax region replesentBlock start=/```/ end=/```/ contains=@Scala
```

Comment:
This is awesome. Would it be difficult to add rules for the other commands (from [separators](https://github.com/marconilanna/REPLesent/#separators-and-delimiters) to [rulers](https://github.com/marconilanna/REPLesent/#horizontal-rulers)). Even just making all of them bold, for instance, would already be great.